### PR TITLE
feat(DENG-913): added dau check for focus_android (release, beta, and nightly channels)

### PR DESCRIPTION
**Pull request by kik-kik**
# feat(DENG-913): added dau check for focus_android (release, beta, and nightly channels)

### CODE REVIEW

**Review by lucia-vargas-a — COMMENTED** [review]
Does this return a result per channel? The reason I am asking is because the KPI is built on the total (not only release channel) so the check would need to return the comparison for Focus Android for all channels.

File: dim_checks/mozdata/focus_android/active_users_aggregates/dim_checks.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  slack_alerts:
+    enabled: true
+    notification_level: INFO
```
[the remark this answers is not in the document]
Comment (kik-kik) [reply dim_checks/mozdata/focus_android/active_users_aggregates/dim_checks.yaml:7]: we also accept `ERROR` which is the default. Is set to that we only send out notifications for checks that failed.

**Comment by kik-kik** [comment]
oh ok, I see. In that case I can tweak the check to do a sum of all channels.

**Comment by kik-kik** [comment]
@lucia-vargas-a made the change. Please let me know if you have any other comments.

File: dim_checks/mozdata/focus_android/active_users_aggregates/dim_checks.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          query: |
+            SELECT
+              SUM(dau),
+            FROM `{{ project_id }}.{{ dataset }}.{{ table }}`
+            WHERE submission_date = "{{ params.partition }}"
+        metric_2:
+          name: distinct_client_count
+          query: |
+            WITH base AS (
+              -- releae channel
```
Comment (lucia-vargas-a) [inline dim_checks/mozdata/focus_android/active_users_aggregates/dim_checks.yaml:31]:
```suggestion
              -- release channel
```

**Review by lucia-vargas-a — APPROVED** [review]
Looks good, only a small typo fix.

**Pull request merged**